docs/index.rst

```diff
@@ -6,7 +6,7 @@
 EasyBuild documentation
 =======================
 
-Welcome to the documentation of `EasyBuild <https://hpcugent.github.io/easybuild>`_, a software build and installation framework
+Welcome to the documentation of `EasyBuild <http://easybuilders.github.io/easybuild>`_, a software build and installation framework
```

Comment:
I was under the impression that https://easybuilders.github.io/easybuild/ just redirect to http://easybuilders.github.io/easybuild/, which is (of course) not the case...

Fixed in 233d423